script/utils/DeployHelper.sol

```diff
@@ -160,7 +172,112 @@ contract DeployHelper is
         }
     }
 
-    function _deployPeripheryContracts() private {
+    function _deployAndConfigStoryNftContracts(
```

Comment:
@kingster-will The deploy logic for Story NFT related contracts are here in `DeployHelper`. `StoryNFT.s.sol` just calls this function.

Reply:
Resolved in [c644e86](https://github.com/storyprotocol/protocol-periphery-v1/pull/87/commits/c644e8656d81fa0b132d160d6ea5aacad3dcaf4a) :)